Octokit.Tests/Reactive/ObservableIssueCommentsClientTests.cs

```diff
@@ -89,10 +95,7 @@ public void RequestsCorrectUrlWithApiOptions()
 
                 client.GetAllForRepository("fake", "repo", options);
 
-                gitHubClient.Connection.Received(1).Get<List<IssueComment>>(
-                    new Uri("repos/fake/repo/issues/comments", UriKind.Relative), 
-                    Arg.Any<Dictionary<string, string>>(),
-                    "application/vnd.github.squirrel-girl-preview");
+                gitHubClient.Received().Issue.Comment.GetAllForRepository("fake", "repo", request, options);
```

Comment:
These tests aren't right.  They arent failing but they also aren't correct.  It may be something to do with the multiple nested subclients on the end of the `.Received()` call

but this observable method doesnt actually call into the `IGitHubClient` method, it calls the `_connection.GetAndFlattenPages()` method and thus this test should be checking for `gitHubClient.Connection.Received(1).Get....`